src/main/java/jenkins/plugins/slack/SlackNotifier.java

```diff
@@ -137,6 +142,7 @@ public SlackNotifier(final String teamDomain, final String authToken, final Stri
         super();
         this.teamDomain = teamDomain;
         this.authToken = authToken;
+        this.authTokenCredentialId = authTokenCredentialId.trim();
```

Comment:
I made a comment on this PR about why I'm doing this.  I don't know how to link to comments.